__tests__/endpoint.test.js

```diff
@@ -81,14 +81,67 @@ describe("/api/articles/", () => {
       });
   });
 });
+describe("/api/articles/:article_id/comments", () => {
+  test("The endpoint will respond with an array of comments from the given article ID", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        body.comments.forEach((comment) => {
+          expect(typeof comment.comment_id).toBe("number");
+          expect(typeof comment.votes).toBe("number");
+          expect(typeof comment.created_at).toBe("string");
+          expect(typeof comment.author).toBe("string");
+          expect(typeof comment.body).toBe("string");
+          expect(typeof comment.article_id).toBe("number");
+        });
+      });
+  });
+
+  test("The endpoint will respond with array", () => {
+    return request(app)
+      .get("/api/articles/9/comments")
+      .expect(200)
+      .then(({ body }) => {
+        expect(Array.isArray(body.comments)).toBe(true);
+      });
+  });
+
+  test("404 given a bad article ID the endpoint will respond with an error", () => {
```

Comment:
I would suggest being more specific than `bad article ID` in your test description. This ID isn't really bad as it could hypothetically exist at some point in the future when your site has lots more articles posted on it. Perhaps go with `article ID doesn't exist in the database` instead